build.gradle.kts

```diff
@@ -2,6 +2,7 @@ plugins {
     java
     id("org.springframework.boot") version "3.4.0"
     id("io.spring.dependency-management") version "1.1.6"
+    id("org.flywaydb.flyway") version "11.0.0"
```

Comment:
Why exactly do we need db migrations?

Reply:
If we ever change our database layout by, for example, renaming a column or table or changing the location a relationship is declared, Spring will likely not be able to figure out the relationship between the old structure and the new one, since that is domain knowledge only we have.
By generating migrations ahead of time instead of at runtime and reviewing (and potentially adjusting) them, we can leverage this domain knowledge to improve the behavior of our migrations, reducing the risk of data loss.

Naturally, the example of renaming is only one example where this may be an issue.